nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -430,6 +450,9 @@ StaticLayer::updateCosts(
       }
     }
   }
+  if (footprint_clearing_enabled_) {
+    setConvexPolygonCost(transformed_footprint_, nav2_costmap_2d::FREE_SPACE);
+  }
```

Comment:
I don't know much about the costmap code, is there a difference to placing this snippet here vs before the setting of the costs just above?

Reply:
I think there is probably correct - else it could be overrided by the update